R/transformCounts.R

```diff
@@ -98,6 +98,14 @@
 #' log2 = log2(x)}
 #' where \eqn{x} is a single value of data.
 #'
+#' \item 'difference': Pairwise differences between features.
+#' Calculates \eqn{x - y} for all unique feature pairs across samples,
+#' where \eqn{x} and \eqn{y} are relative abundances or transformed values.
```

Comment:
x and y could be anything, relative abundances do not have particular precedence here - so this could be rather something like "where x and y are entries of the specified assay.type"